lib/storage.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
This looks great!  Consider adding an if statement to check whether the schema directory already exists and if not creates the directory before attempting to create, update, delete, etc. files inside of it.  This isn't necessary for the assignment, but something good to consider :)